cli/command/deploy.go

```diff
@@ -2,7 +2,16 @@ package command
 
 import "github.com/urfave/cli"
 
-func (f *CommandFactory) Deploy() cli.Command {
+type DeployCommand struct {
+	*CommandMediator
+}
+
+func NewDeploy(m *CommandMediator) cli.Command {
```

Comment:
We should follow best practices and have a proper NewDeployCommand constructor. Fetching the cli.Command should be a method on the struct (was deployCommand.Command(), previously).